.teamcity/_self/projects/WebApp.kt

```diff
@@ -753,305 +753,157 @@ object CheckCodeStyleBranch : BuildType({
+					# Run the test
+					export VIEWPORT_NAME=$viewportName
+					export LOCALE=en
+					export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
+					export DEBUG=pw:api
 
-				xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --testNamePattern @parallel --maxWorkers=%E2E_WORKERS% specs/specs-playwright
-			""".trimIndent()
-			dockerImage = "%docker_image_e2e%"
-			dockerRunParameters = "-u %env.UID% --security-opt seccomp=.teamcity/docker-seccomp.json --shm-size=8gb"
-		}
-		bashNodeScript {
-			name = "Collect results"
-			executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
-			scriptContent = """
-				set -x
+					xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --testNamePattern @parallel --maxWorkers=%E2E_WORKERS% specs/specs-playwright
+				""".trimIndent()
+				dockerImage = "%docker_image_e2e%"
+				dockerRunParameters = "-u %env.UID% --security-opt seccomp=.teamcity/docker-seccomp.json --shm-size=8gb"
+			}
+			bashNodeScript {
+				name = "Collect results"
+				executionMode = BuildStep.ExecutionMode.RUN_ON_FAILURE
+				scriptContent = """
+					set -x
 
-				mkdir -p screenshots
-				find test/e2e/results -type f -path '*/screenshots/*' -print0 | xargs -r -0 mv -t screenshots
+					mkdir -p screenshots
+					find test/e2e/results -type f -path '*/screenshots/*' -print0 | xargs -r -0 mv -t screenshots
 
-				mkdir -p logs
-				find test/e2e/results -name '*.log' -print0 | xargs -r -0 tar cvfz logs.tgz
-			""".trimIndent()
-			dockerImage = "%docker_image_e2e%"
+					mkdir -p logs
+					find test/e2e/results -name '*.log' -print0 | xargs -r -0 tar cvfz logs.tgz
+				""".trimIndent()
+				dockerImage = "%docker_image_e2e%"
+			}
 		}
-	}
 
-	features {
-		perfmon {
-		}
-		pullRequests {
-			vcsRootExtId = "${Settings.WpCalypso.id}"
-			provider = github {
-				authType = token {
-					token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
+		features {
+			perfmon {
+			}
+			pullRequests {
+				vcsRootExtId = "${Settings.WpCalypso.id}"
+				provider = github {
+					authType = token {
+						token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
+					}
+					filterAuthorRole = PullRequests.GitHubRoleFilter.EVERYBODY
 				}
-				filterAuthorRole = PullRequests.GitHubRoleFilter.EVERYBODY
 			}
-		}
-		commitStatusPublisher {
-			vcsRootExtId = "${Settings.WpCalypso.id}"
-			publisher = github {
-				githubUrl = "https://api.github.com"
-				authType = personalToken {
-					token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
+			commitStatusPublisher {
+				vcsRootExtId = "${Settings.WpCalypso.id}"
+				publisher = github {
+					githubUrl = "https://api.github.com"
+					authType = personalToken {
+						token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
+					}
 				}
 			}
 		}
-	}
 
-	triggers {
-		vcs {
-			branchFilter = """
-				+:trunk
-			""".trimIndent()
+		triggers {
+			vcs {
+				branchFilter = """
+					$filter
+				""".trimIndent()
```

Comment:
I think this can be replaced by a single line
```suggestion
				branchFilter = filter
```

Reply:
Changed.